octobot_commons/aiohttp_util.py

```diff
@@ -14,6 +14,18 @@
 #
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
+import ssl
+
+import contextlib
+import aiohttp
+import certifi
+
+import octobot_commons.logging
+import octobot_commons.constants
+
+DEFAULT_TEST_URL = (
```

Comment:
Should we move it in tests as it is only used in tests ?

Reply:
It's used by octobot and tentacles manager as well